source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -300,6 +302,13 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
     [serviceSend],
   );
 
+  const setMapLayersToRefresh = useCallback(
+    (refreshLayers: ILayerItem[]) => {
+      serviceSend({ type: 'SET_REFRESH_MAP_LAYERS', refreshLayers });
+    },
+    [serviceSend],
+  );
```

Comment:
hmm - I'm not seeing a corresponding `SET_REFRESH_MAP_LAYERS` state on mapMachine.ts - is this correct?